site_scons/SGppConfigure.py

```diff
@@ -554,8 +555,6 @@ def configureClangCompiler(config):
   #    "If you are going to use optimisations turned on with gcc > 4.0 (for example -O2),
   #     ensure you also compile with -fno-strict-aliasing"
 #   if not config.env["USE_HPX"]:
-  config.env.Append(CPPFLAGS=["-fopenmp=libiomp5"])
-  config.env.Append(LINKFLAGS=["-fopenmp=libiomp5"])
```

Comment:
According to 0ee88be6 (`git blame`), this was to reduce dependencies as `libiomp5` was included with Clang.

Reply:
conda-forge uses clang 4.0.1, and apparently libiomp5 is disabled with this compiler, as it complains that omp.h is not found with this flag enabled, maybe the gomp variant is enabled instead ? nevertheless this option is still too specific.

Reply:
Huh. On our 16.04.6 LTS machine, it doesn't need the `=libiomp5` (if you use Clang from the Ubuntu packages, it's probably different for Clang's binary distribution). So we can remove that. You do need `-fopenmp` of course, please add:
```suggestion
  config.env.Append(CPPFLAGS=["-fopenmp"])
  config.env.Append(LINKFLAGS=["-fopenmp"])
```